jekyll/_cci2/concepts.md

```diff
@@ -219,6 +207,8 @@ Each separate job defined within your configuration runs in a unique execution e
 
 An _image_ is a packaged system that includes instructions for creating a running container or virtual machine, and you can define an image for each executor. CircleCI provides a range of images for use with the Docker executor, called _convenience images_ (details in the [images](#images) section).
 
+{% include snippets/docker-auth.md %}
```

Comment:
Yeah so I added a few here just because it's a reference page, so people less likely to read more than once section, to try and catch people in relevant sections.

Also RE not adding the banner to all pages, I just felt that for some pages the recommendation is so out of the scope of what's being discussed that it would seem out of place. Happy to add it in to those pages though if you think that would be better.

Reply:
All good points!